app/mailers/spree/user_mailer.rb

```diff
@@ -3,13 +3,13 @@ class UserMailer < BaseMailer
     def reset_password_instructions(user, token, *args)
       @edit_password_reset_url = spree.edit_spree_user_password_url(:reset_password_token => token, :host => Spree::Store.current.url)
 
-      mail to: user.email, from: from_address, subject: Spree::Store.current.name + ' ' + I18n.t(:subject, :scope => [:devise, :mailer, :reset_password_instructions])
+      mail to: user.email, from: from_address(Spree::Store.current), subject: Spree::Store.current.name + ' ' + I18n.t(:subject, :scope => [:devise, :mailer, :reset_password_instructions])
```

Comment:
Sorry to tack improvements into a fix PR, but could we change this to 

```
@store = Spree::Store.current

mail to: user.email, from: from_address(@store), subject: @store.name + ' ' + I18n.t(:subject, :scope => [:devise, :mailer, :reset_password_instructions])
```

Eventually, we'll want to be able to get the correct store to send this email as, rather than just the default store, this would be a step towards that.